Library/Homebrew/extend/os/mac/extend/ENV/super.rb

```diff
@@ -96,6 +96,15 @@ def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_a
     self["HOMEBREW_DEVELOPER_DIR"] = if is_xcode_sdk
       MacOS::Xcode.prefix.to_s
     else
```

Comment:
Probably have to handle this condition when `MacOS.active_developer_dir` is CLT as `m4` would still be broken